apps/state/lib/state/routes_by_service.ex

```diff
@@ -45,7 +45,7 @@ defmodule State.RoutesByService do
         []
 
       items ->
-        Enum.flat_map(items, fn {_key, routes} -> routes end)
+        Enum.uniq(Enum.flat_map(items, fn {_key, routes} -> routes end))
```

Comment:
Can you add back the `uniq` step when creating the ETS data? Doing that makes this lookup 250x faster:
```
## State.RouteUniqBench
benchmark name       iterations   average time
flat_map dedup         10000000   0.13 µs/op
uniq flat_map dedup    10000000   0.38 µs/op
flat_map ets              50000   34.25 µs/op
uniq flat_map ets         10000   101.70 µs/op
```